cmp/dataview/DataViewModel.js

```diff
@@ -32,23 +32,26 @@ export class DataViewModel {
     };
 
     /**
+     * @param {function} [contextMenuFn] - closure returning a StoreContextMenu().
+     * @param {string} emptyText - empty text to display if DataView has no records. Can be valid HTML.
      * @param {function} itemFactory - elemFactory for the component used to render each item.
      *      Will receive record via its props.
-     * @param {BaseStore} store - store containing the data for the dataview.
      * @param {(StoreSelectionModel|Object|String)} [selModel] - selection model to use,
      *      config to create one, or 'mode' property for a selection model.
-     * @param {function} [contextMenuFn] - closure returning a StoreContextMenu().
+     * @param {BaseStore} store - store containing the data for the dataview.
```

Comment:
This change of course looks fine, but why are we reordering the arguments in the doc string?  Seems like our practice has been that the most important, required arguments go first, and the more secondary (and often optional) arguments follow.  Are you suggesting we go to an alphabetical model?

Reply:
Exactly..  for components with a very small number of configs order them alphabetically, and components with a lot of configs we can pin the most important at the top.
What do you think?

Reply:
I think we need one rule or another -- how do you determine "a lot of configs".  Let's stick with the existing practice of ordering by importance.